src/main/java/net/fabricmc/loader/gui/FabricGuiEntry.java

```diff
@@ -42,7 +41,7 @@ private static void openWindow(FabricStatusTree tree, boolean shouldWait) throws
 	/** @param exitAfter If true then this will call {@link System#exit(int)} after showing the gui, otherwise this will
 	 *            return normally. */
 	public static void displayCriticalError(Throwable exception, boolean exitAfter) {
-		exception.printStackTrace(System.err);
+		FabricLoader.INSTANCE.getLogger().fatal("", exception);
```

Comment:
This should include some message along the lines of "A critical error occurred", not ""